Sources/MenuAnimator.swift

```diff
@@ -122,8 +122,7 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
     private var panRecognizer: UIPanGestureRecognizer!
```

Comment:
@okonor please make this and some other properties that are force-unwrapped as expliced optionals (`?`).
Actually, we're trying not to use `!` in a codebase since it can be lead to crash without clear description what's wrong.